### ISSUE TITLE
Gray out overlapping not centered when zoom below 1

### ISSUE THREAD

**Issue by molivier-fcms** [koreader/koreader#11219]
* KOReader version: 2023.10
* Device: Kobo; Ubuntu

#### Issue

I think this is related to issue #9967. When zoom is below 1 with pdf, the text was stuck on left side of the screen.

It has been corrected. But it seems page overlapping by grey out behave the same way : I can clearly see a kind of grey box stuck on left side as well.

![box](https://github.com/koreader/koreader/assets/39309715/4bc6274d-4ce5-44f0-80b0-c04348d33f7c)

**Comment by NiLuJe** [comment]
Yeah, the overlap thing is a nightmare; I tried to deal with a few obviously broken cases, but I don't use it enough (read: at all) to warrant suffering through it all.
